src/naive/date.rs

```diff
@@ -115,6 +115,19 @@ impl NaiveWeek {
     }
 }
 
+/// A duration in calendar days. This is useful becuase when using `TimeDelta` it is possible
```

Comment:
I think only the first sentence should be on the first line. Also "becuase", should backtick-quote `TimeDelta::days(1)`. And it's weird to be talking about `TimeDelta` here because that doesn't actually exist in this branch.

If we're going with `TimeDelta` as an output type only, we probably shouldn't have constructors like `TimeDelta::days()`?

Reply:
Yep looks like I got ahead of myself with the docs here. I'll leave the comparison to `Duration::days` for now so that users can figure out when to use each. 

`TimeDelta` would still be an output type when using the `signed_duration_since` and equivalent operations Ideally though, we would make `Days` be the output type for `NaiveDate::signed_duration_since`, however `Days` doesn't currently allow negative, so this is something to think about. Overall we probably want to be consistent in allowing signed duration everywhere or nowhere, but perhaps it is also possible to have the calendar durations (`Months` and `Days`) not support negative, and the precise durations (now `Duration`, later potentially `TimeDelta`) support fowards and backwards.

If we don't want to support signed calendar duration (my preference), then we could change `NaiveDate::signed_duraiton_since` to `NaiveDate::days_since` or `NaiveDate::absolute_days_since` or equivalent (and also have `NaiveDate::months_since`).

Reply:
>  but I think that's maybe a feature more than a bug, as it's actually the tricky case?

This sounds excellent to me. IMO using a negative duration as an input would usually provide unexpected results. This also allows unfallible versions returning `TimeDelta` and equivalent fallible ones returning `Duration`